### PR TITLE
[Release] 2.15 beta 1

### PR DESCRIPTION
**Pull request by abelgardep**
Implements #2843 
Needs https://github.com/owncloud/android-library/pull/316

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+## 2.15 beta v1 (May 2020)
+- Login rearchitecture
+- OpenId Connect implementation
```
Comment (davigonz) [inline CHANGELOG.md:3]: I would write `Support for OpenId Connect`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+- Biometrical lock
+- UI improvements, including:
+  + New bottom navigation bar
+- End of support for Android KitKat
+- End of support for older than oc10 servers
+- Bug fixes, including:
+  + Fix crash when changing orientation in some operations
+  + Fix OAuth2 token is not renewed after being revoked
+  + Support for usernames with '+'
```
Comment (davigonz) [inline CHANGELOG.md:12]: Maybe this one is not a bug fix

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,16 @@
+## 2.15 beta v1 (May 2020)
+- Login rearchitecture
+- OpenId Connect implementation
+- Biometrical lock
+- UI improvements, including:
+  + New bottom navigation bar
+- End of support for Android KitKat
+- End of support for older than oc10 servers
```
Comment (davigonz) [inline CHANGELOG.md:8]: `End of support for servers older than 10 version`

**Comment by jesmrec** [comment]
QA:

- [X] Basic authentication 
- [X] OAuth2 authentication & renewal
- [X] OIDC authentication & renewal
- [x] Basic operations
- [X] Biometrical lock
- [X] Upgrade from 2.14.2 (regular)
- [x] Upgrade from 2.14.2 (older than 10 server)
- [X] Upgrade from 2.14.2 (Android 4.4)
  
Approved on my side and expecting tons of feedback!

**Pull request merged**